Spigot-Server-Patches/0581-Improve-ServerGUI.patch

```diff
@@ -0,0 +1,453 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: AlexProgrammerDE <40795980+AlexProgrammerDE@users.noreply.github.com>
+Date: Sat, 3 Oct 2020 08:27:40 +0200
+Subject: [PATCH] Improve ServerGUI
+
+- Added logo to server frame
+- Show tps in the server stats
+- Fix title issue on gnome
+
+diff --git a/src/main/java/com/destroystokyo/paper/gui/RAMDetails.java b/src/main/java/com/destroystokyo/paper/gui/RAMDetails.java
+index e463a86a6ad6c4f2885f4df159411bad294e1208..50960535a64efaf9dfb9bb95ea765b6c19616b6f 100644
+--- a/src/main/java/com/destroystokyo/paper/gui/RAMDetails.java
++++ b/src/main/java/com/destroystokyo/paper/gui/RAMDetails.java
+@@ -2,8 +2,6 @@ package com.destroystokyo.paper.gui;
+ 
+ import net.minecraft.server.MinecraftServer;
+ import net.minecraft.server.SystemUtils;
+-import org.bukkit.Bukkit;
+-
+ import javax.swing.DefaultListCellRenderer;
+ import javax.swing.DefaultListSelectionModel;
+ import javax.swing.JList;
+@@ -58,9 +56,20 @@ public class RAMDetails extends JList<String> {
+     public void update() {
+         GraphData data = RAMGraph.DATA.peekLast();
+         Vector<String> vector = new Vector<>();
++
++        // Paper start - Add tps entry
++        double[] tps = org.bukkit.Bukkit.getTPS();
```

Comment:
`Bukkit` is imported.

Reply:
yeah idk what exactly happened there. Imma revert it